packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -70,16 +70,19 @@ const insertBandwidthLimit = (sdpLines, index) => {
  * @param {String} [level=QUALITY_LEVELS.HIGH] quality level for max-fs
  * @returns {String}
  */
-const setMaxFs = (sdp, level = QUALITY_LEVELS.HIGH) => {
-  if (!MAX_FRAMESIZES[level]) {
-    throw new ParameterError(`setMaxFs: unable to set max framesize, value for level "${level}" is not defined`);
+const setRemoteVideoConstraints = (sdp, level = QUALITY_LEVELS.HIGH) => {
+  const maxFs = REMOTE_VIDEO_CONSTRAINTS.MAX_FS[level];
+  const maxMbps = REMOTE_VIDEO_CONSTRAINTS.MAX_BITRATE[level];
+
+  if (!maxFs || !maxMbps) {
+    throw new ParameterError(`setRemoteVideoConstraints: unable to set max framesize, value for level "${level}" is not defined`);
```

Comment:
this message needs to be updated to include max-mbps and would be useful if it said which one of the 2 is missing